src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/components/AddFeatureFlag.tsx

```diff
@@ -33,23 +33,26 @@ export const AddFeatureFlag = () => {
   const router = useRouter();
 
   const handleSubmit = (
-    event: FormEventHandler<HTMLFormElement> & {
+    event: FormEvent<HTMLFormElement> & {
       target: FeatureFlagElement;
     }
   ) => {
     event.preventDefault();
 
+    const eventTarget: FeatureFlagElement = event.target;
+
+    console.debug(eventTarget);
+
     const data = {
-      name: event.target.name.value,
-      isEnabled: event.target.isEnabled ? true : false,
+      name: eventTarget.name.value,
+      isEnabled: eventTarget.isEnabled.checked ? true : false,
       dependencies: event.target.dependencies.value.split(","),
       allowList: event.target.allowList.value.split(","),
       waitList: event.target.waitList.value.split(","),
       owner: event.target.owner.value,
```

Comment:
Q: Should these be updated to use `eventTarget.` instead of `event.target.` too?
```suggestion
      dependencies: eventTarget.dependencies.value.split(","),
      allowList: eventTarget.allowList.value.split(","),
      waitList: eventTarget.waitList.value.split(","),
      owner: eventTarget.owner.value,
```